src/index.html

```diff
@@ -12,7 +15,39 @@
       href="styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page__body">
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the class name `calendar--start-day-sun` suggests that it's a modifier, but it's unclear what value it modifies. Ensure that the class name follows the BEM naming convention and accurately describes its purpose.

Reply:
O nome segue a nomenclatura BEM

Reply:
sim